assets/onboarding/index.jsx

```diff
@@ -1,24 +1,28 @@
-import { Stepper } from '@woocommerce/components';
 import { render } from '@wordpress/element';
-import { useFullScreen } from './use-fullscreen';
+import { useFullScreen } from '../react-hooks';
 import { steps } from './steps.jsx';
+import { QueryStringRouter } from './query-string-router';
+import Navigation from './navigation';
+import ContentContainer from './content-container';
 
-function SenseiOnboardingPage() {
-	useFullScreen( [ 'sensei-color', 'sensei-onboarding__page' ] );
+/**
+ * Query string name used to route the onboarding wizard.
+ */
+const QUERY_STRING_NAME = 'onboarding-step';
 
-	const currentStep = steps[ 0 ];
+const SenseiOnboardingPage = () => {
```

Comment:
@yscik WDYT about we start using the functions always as arrow functions? I like it because we use the modern format and it makes easier some cases when we want to return some value directly. Also, it helps us using the `this` from some scopes.
But if you have a different opinion we can rollback ;)

Reply:
Works for me. I usually preferred the function form, because it actually gives the function a name that's displayed in stack traces and such, but I just checked and that also works with this syntax now by some magic.